packages/styles/src/palettes/_mixins.scss

```diff
@@ -13,6 +13,7 @@
 @mixin palettes($theme) {
   body {
     background-color: var(--post-current-palette-bg) !important;
+    color: var(--post-current-palette-fg) !important;
```

Comment:
The `--post-current-palette-fg` is for the `.palette-text` utility which allows to have blue headings on cargo green. The body should not use it.

Feel free to rename the custom property if you think it's unclear!

```suggestion
```